packages/ag-charts-enterprise/src/main.ts

```diff
@@ -60,19 +60,11 @@ _ModuleSupport.registerModule(TreemapSeriesModule);
 _ModuleSupport.registerModule(WaterfallModule);
 _ModuleSupport.registerModule(ZoomModule);
 
-export class AgEnterpriseCharts {
-    public static create(options: AgChartOptions): AgChartInstance {
+export class AgChart extends AgChartCommunity {
```

Comment:
Does this approach work for the FW cases, i.e. do we currently get the license message on opening an enterprise React example?

Reply:
Haven't checked specific libraries, just plain Javascript. I'll verify tomorrow.